src/librustdoc/html/format.rs

```diff
@@ -732,9 +732,9 @@ fn fmt_type<'cx>(
 
     match *t {
         clean::Generic(name) => write!(f, "{}", name),
-        clean::ResolvedPath { did, ref path, is_generic } => {
+        clean::ResolvedPath { did, ref path } => {
             // Paths like `T::Output` and `Self::Output` should be rendered with all segments.
-            resolved_path(f, did, path, is_generic, use_absolute, cx)
+            resolved_path(f, did, path, path.is_generic(), use_absolute, cx)
```

Comment:
You're looking at an old diff; I changed `path.is_generic()` to `false` in a later commit (the one we're nervous about ;).

Reply:
I should probably remove the comment above this call too.

Reply:
I have since dropped the commit that removed `is_generic()`, so this place now has `resolved_path(..., path.is_assoc_ty())`. I think `resolved_path` and related functions need to be cleaned up a lot in general, but I think let's do that in a different PR so this one doesn't keep growing ;)

Reply:
Actually, it looks like I should be able to pretty easily clean up `resolved_path` such that the argument that `path.is_generic()` is used for is removed!

Reply:
... or not; it looks like there's a single place where `use_absolute` is not `false` and not `None` :/